### PR TITLE
A-T auth0 config changes

### PR DESCRIPTION
**Pull request by ssettipalli**

### CODE REVIEW

File: study-builder/tenants/atcp/emailTemplates/verify_email.html [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+            <td
+              style="
+                border-collapse: collapse;
+                text-align: right;
+                float: right;
+                padding: 30px 40px 15px 0;
+              "
+            >
+              <img
+                src="https://storage.googleapis.com/dev-at-assets/email-logo.png"
```
Comment (yufengwng) [inline study-builder/tenants/atcp/emailTemplates/verify_email.html]: Are we sure we want to hardcode the dev bucket?

File: study-builder/tenants/atcp/emailTemplates/verify_email.html [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+      >
+        <tbody>
+          <tr>
+            <td
+              style="border-collapse: collapse"
+              align="center"
+              valign="middle"
+            >
+              <a
+                href="https://atcp-dot-broad-ddp-dev.uc.r.appspot.com/"
```
Comment (yufengwng) [inline study-builder/tenants/atcp/emailTemplates/verify_email.html]: Need to remove hardcode.

File: study-builder/tenants/atcp/pages/login.html [hunk 3]
```diff
@@ -0,0 +1,502 @@
...
+    <title data-translate="title"></title>
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0"
+    />
+    <link
+      id="app-favicon"
+      rel="icon"
+      type="image/x-icon"
+      href="https://atcp-dot-broad-ddp-dev.uc.r.appspot.com/favicon.ico"
```
Comment (yufengwng) [inline study-builder/tenants/atcp/pages/login.html]: Watch out for hardcode!

File: study-builder/tenants/atcp/pages/password_reset.html [hunk 4]
```diff
@@ -0,0 +1,342 @@
...
+          href="https://atcp-dot-broad-ddp-dev.uc.r.appspot.com/favicon.ico">
+    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <!-- Latest compiled and minified JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <link href="https://fonts.googleapis.com/css?family=Lato:400,700,700italic,400italic,300,300italic" rel="stylesheet"
+          type="text/css">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://atcp-dot-broad-ddp-dev.uc.r.appspot.com/assets/auth0-styles/main.css">
```
Comment (yufengwng) [inline study-builder/tenants/atcp/pages/password_reset.html]: Hardcode!

File: study-builder/tenants/atcp/rules/Register User in Pepper.js [hunk 5]
```diff
@@ -0,0 +1,228 @@
+function (user, context, callback) {
```
Comment (yufengwng) [inline study-builder/tenants/atcp/rules/Register User in Pepper.js]: What's the difference between this file and the one in `tenants/register-user-in-pepper.js`? I think it's better if we just have one copy of this auth0 rule..

File: study-builder/tenants/atcp/tenant.yaml [hunk 6]
```diff
@@ -0,0 +1,385 @@
...
+    html: ./pages/login.html
+  - name: password_reset
+    enabled: true
+    html: ./pages/password_reset.html
+resourceServers: []
+clients:
+  - name: AT-Dev
+    allowed_clients: []
+    allowed_logout_urls:
+      - 'http://localhost:4200'
```
Comment (yufengwng) [inline study-builder/tenants/atcp/tenant.yaml]: Please replace these URLs.

File: study-builder/tenants/atcp/tenant.yaml [hunk 7]
```diff
@@ -0,0 +1,385 @@
...
+    jwt_configuration:
+      alg: RS256
+      lifetime_in_seconds: 36000
+      secret_encoded: false
+    oidc_conformant: true
+    refresh_token:
+      rotation_type: non-rotating
+      expiration_type: non-expiring
+    sso_disabled: false
+databases:
```
Comment (yufengwng) [inline study-builder/tenants/atcp/tenant.yaml]: You don't need to include config for a bunch of these... they should come as default for the tenant.

File: study-builder/tenants/atcp/tenant.yaml [hunk 8]
```diff
@@ -0,0 +1,385 @@
...
+  support_url: ''
+  universal_login:
+    colors:
+      page_background: '#000000'
+      primary: '#ea5323'
+emailProvider:
+  name: sendgrid
+  credentials:
+    api_key: '##SENDGRID_API_KEY##'
+  default_from_address: atcp@dev.datadonationplatform.org
```
Comment (yufengwng) [inline study-builder/tenants/atcp/tenant.yaml]: Dev?

File: study-builder/config/atcp-config.json.ctmpl [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+    "LOCALHOST_LOGOUTS": ",http://localhost:4200, http://localhost:4200/session-expired, http://localhost:4200/account-activation-required, http://localhost:4200/join-us",
+    "LOCALHOST_ORIGINS": ",http://localhost:4200",
+  {{else}}
+    "LOCALHOST_CALLBACKS": "",
+    "LOCALHOST_LOGOUTS": "",
+    "LOCALHOST_ORIGINS": "",
+  {{end}}
+    "BASE_URL": "{{$conf.baseWebUrl}}",
+    "LOGIN_DOMAIN": "{{$conf.auth0.loginDomain}}",
+    "STUDY_EMAIL": "atcp@{{$environment}}.datadonationplatform.org",
```
Comment (yufengwng) [inline study-builder/config/atcp-config.json.ctmpl]: What about prod?

File: study-builder/tenants/atcp/pages/password_reset.html [hunk 10]
```diff
@@ -0,0 +1,342 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <title data-translate="title"></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0"/>
+    <link id="app-favicon" rel="icon" type="image/x-icon"
+          href="{{ url }}/favicon.ico">
```
Comment (yufengwng) [inline study-builder/tenants/atcp/pages/password_reset.html]: The password reset page doesn't have a `{{url}}` variable... did you mean `##BASE_URL##`?

Comment (ssettipalli) [reply study-builder/tenants/atcp/pages/password_reset.html]: yes. will update it

File: study-builder/config/atcp-config.json.ctmpl (hunk 9 above)
Comment (ssettipalli) [reply study-builder/config/atcp-config.json.ctmpl]: need to check and will update code if needed

**Review by yufengwng — APPROVED** [review]

**Pull request merged**